packages/cli/src/load-nodes-and-credentials.ts

```diff
@@ -376,7 +376,7 @@ export class LoadNodesAndCredentials {
 				loader.reset();
 				await loader.loadAll();
 				await this.postProcessLoaders();
-				push.broadcast('nodeDescriptionUpdated');
+				push.broadcast('nodeDescriptionUpdated', {});
```

Comment:
i don't know how to make the second argument conditional when the first argument is of this type.
perhaps we should change the signature on these functions to accept a single object arg with `type` and `data`, instead of 2 args.